lib/errors/AsserterError.js

```diff
@@ -1,4 +1,5 @@
 /**
+ * @license
  * Copyright (c) 2020 DigiByte Foundation NZ Limited
```

Comment:
We will need to open a future PR to update the License headers throughout to the same header we use in the core protocol.